test/test-fs.c

```diff
@@ -427,6 +427,28 @@ static void rmdir_cb(uv_fs_t* req) {
 }
 
 
+static void assert_is_file_type(uv_dirent_t dent) {
+  #ifdef HAVE_DIRENT_TYPES
+    /*
+     * For Apple and Windows, we know getdents is expected to work but for other
+     * environments, the filesystem dictates whether or not getdents supports
+     * returning the file type.
+     *
+     *   See:
+     *     http://man7.org/linux/man-pages/man2/getdents.2.html
+     *     https://github.com/libuv/libuv/issues/501
+     */
+    #if defined(__APPLE__) || defined(_WIN32)
+      ASSERT(dent.type == UV_DIRENT_FILE);
+    #else
+      ASSERT(dent.type == UV_DIRENT_FILE || dent.type == UV_DIRENT_UNKNOWN);
+    #endif
+  #else
+      ASSERT(dent.type == UV_DIRENT_UNKNOWN);
```

Comment:
That was a relic of moving this from one of the places with an extra level of indentation.  ;)  I'll update the PR.